src/psyclone/tests/domain/gocean/transformations/gocean1p0_transformations_test.py

```diff
@@ -886,9 +886,15 @@ def test_go_omp_loop_applied_to_non_loop():
 
     # Attempt to (erroneously) apply the GO OMP Loop transformation
     # to the first node in the schedule (which is now itself an
-    # OMP Loop transformation)
-    with pytest.raises(TransformationError):
+    # OMPDoDirective).
+    with pytest.raises(TransformationError) as err:
+        _, _ = ompl.validate(omp_schedule.children[0])
```

Comment:
I don't think validate returns anything so you should not have `_, _ = `

Reply:
Oops.